src/base/static/js/views/layer-view.js

```diff
@@ -59,6 +62,14 @@
         geom = this.model.get('geometry');
         if (geom.type === 'Point') {
           this.latLng = L.latLng(geom.coordinates[1], geom.coordinates[0]);
+
+          // If we've saved an icon url in the model, use that
+          // TODO: this could be refactored into style rule resolution in
```

Comment:
#705 changes the way we handle style/zoom rules, and actually does away with the resolution code in L.argo. We can handle `iconUrl` resolution there.